README.md

```diff
@@ -22,6 +22,11 @@ manager behavior.
 - Resolves references with the `bal:///` prefix to data from a
   pre-configured library of assets stored in a `.json` file.
 
+- Environment variables are expanded in string-type trait property
+  values (using the `$var` or `${var}` syntax, escape `$` using `$$`).
+  In addition `$bal_library_path` and `$bal_library_dir` can be used
```

Comment:
[Consider] The standard env vars are normally ALL CAPS, perhaps we should follow the lead in docs & implementation.

Reply:
Did you see the note in the commit message? I thought it was worth differentiating implicit BAL vars as they dont actually exist in the environment. Also `BAL_LIBRARY_PATH` is an input env var that BAL uses, and it's value may differ (as `bal_linrary_path` is guaranteed to be absolute)

Reply:
I did not, fair enough :+1: